### DISCUSSION TITLE
About the loss.py

### DISCUSSION THREAD

**Discussion by LonePeng** [ultralytics/yolov5#7270] [Q&A] [answered]
The CIOU_LOSS suits my project better than the Giou_Loss function ,but I don't know how to adjust the loss.py to replace GIOU_Loss.Can someone help me? Thanks very much.

**Comment by glenn-jocher** [comment, marked as answer]
@LonePeng CIoU is already the default regression loss in YOLOv5
https://github.com/ultralytics/yolov5/blob/ea72b84f5e690cb516642ce2d9ae200145b0af34/utils/loss.py#L141-L142

**Reply by LonePeng** [reply]
Thanks.
